### ISSUE TITLE
MSE without the Mean, confusing when comparing to Tensorflow

### ISSUE THREAD

**Issue by GLJeff** [jcjohnson/pytorch-examples#26]
I'd recommend actually using the _mean_ squared error for all these examples. These lines in particular are very misleading to someone not familiar with PyTorch loss function params:
```
# The nn package also contains definitions of popular loss functions; in this
# case we will use Mean Squared Error (MSE) as our loss function.
loss_fn = torch.nn.MSELoss(size_average=False)
```
Calling this the popular MSE loss function is confusing at best. At the very least, you should make note of what the size_average=False parameter is doing, i.e. no longer making it a MSE! That is a SE sum. New and potential PyTorch users are likely using these examples to compare to Tensorflow, and it is extremely easy to become frustrated when your losses which you think are MSE are ending up way larger compared to the same model running in TF. It's also easy to become frustrated when you start hitting NAN values a lot easier with large datasets.

Other than that, I loved the examples. An excellent introduction to an amazing package.

**Comment by jcjohnson** [comment]
Thanks for pointing this out, you are completely correct. I wanted to use a consistent loss function across all examples, and in the "raw" examples I thought it was cleaner to use a sum rather than mean. I added a note about this in https://github.com/jcjohnson/pytorch-examples/commit/73a662bbe9fce257ec8c3cdbb44a8112a42d39f3.
